server/routes/game.controller.ts

```diff
@@ -12,10 +11,23 @@ const getAll = async (req: Request, res: Response) => {
   }
 };
 
+const getGamesById = async (req: Request, res: Response) => {
```

Comment:
Good implementation. Pretty self explanatory and good that you threw in a try/catch.